tidb-cloud/data-service-postman-integration.md

```diff
@@ -0,0 +1,93 @@
+
+        - If you have not logged into Postman, follow the on-screen instructions to log into Postman first.
+        - If you clicked **Run in Postman Desktop**, follow the on-screen instructions to open the Postman desktop app.
+
+    2. On the page of your target workspace in Postman, click **Import** in the left navigation menu.
+    3. Copy the Data App URL in the TiDB Cloud dialog, and then paste the URL to Postman for the import.
+
+5. After you paste the URL, Postman imports the Data App automatically as a new [collection](https://learning.postman.com/docs/collections/collections-overview). The name of the collection is in the `TiDB Data Service - <Your App Name>` format.
+
+    In the collection, the deployed endpoints are grouped in the **Deployed** folder and the undeployed endpoints are grouped in the **Draft** folder.
+
+## Step 2. Configure your Data App API key in Postman
+
+Before running the imported Data App in Postman, you need to configure the API key for the Data App in Postman as follows:
+
+1. In the left navigation menu of Postman, click `TiDB Data Service - <Your App Name>` to open a tab for it on the right side.
+2. Under the `TiDB Data Service - <Your App Name>` tab, click the **Variables** tab.
+3. In the variable table, enter the public key and private key for your Data App in the **Current value** column.
+4. In the upper-right corner of the `TiDB Data Service - <Your App Name>` tab, click **Save**.
+
+## Step 3. Run Data App in Postman
+
+To run your Data App in Postman, take the following steps:
+
+1. In the left navigation pane of Postman, expand the **Deployed** or **Draft** folder, and then click your endpoint name to open a tab for it on the right side.
+2. Under the `<Your Endpoint Name>` tab, you can call your endpoint as follows:
+
+    - For an endpoint without parameters, you can click **Send** to call it directly.
+    - For an endpoint with parameters, you need to fill in the parameter values first, and then click **Send**.
+
+        - For a `GET` or `DELETE` request, fill in the parameter values in the **Query Params** table.
+        - For a `POST` or `PUT` request, click the **Body** tab, and then fill in the parameter values as a JSON object. If **Batch Operation** is enabled for the endpoint in TiDB Cloud Data Service, fill in the parameter values as an array of JSON objects.
+
+3. Check the response in the lower pane.
+
+4. If you want to call the endpoint again with different parameter values, you can edit the parameter values accordingly, and then click **Send** again.
+
+To learn more about the Postman usage, see [Postman documentation](https://learning.postman.com/docs).
+
+## Deal with new changes in Data App
+
+After a Data App is imported to Postman, TiDB Cloud Data Service will not synchronize the following new changes of the Data App to Postman.
```

Comment:
I think it will not automatically import any new changes ! Can you please confirm with @clark Wen ? 
If so then you don't need to make it so obvious in the docs. Also you can emphasis on the word automatic i.e "TiDB Cloud Data Service will not automatically synchronize

Reply:
What about SQL statement updates without parameter changes or SQL endpoint max row changes?
@clark1013 @jusong PTAL